evaluators/rank/FScoreEvaluator/__init__.py

```diff
@@ -0,0 +1,44 @@
+from typing import Sequence, Any, Optional
+
+from jina.executors.evaluators.rank import BaseRankingEvaluator
+
+
+class FScoreEvaluator(BaseRankingEvaluator):
+    """
+    :class:`FScoreEvaluator` Gives the f score of a search system result. (https://en.wikipedia.org/wiki/F-score)
+    :param eval_at: the point at which precision and recall are computed, if None give, will consider all the input to evaluate
+    :param beta: Parameter to weight differently precision and recall. When beta is 1, the fScore corresponds to the harmonic mean
+        of precision and recall
```

Comment:
please add docstring for `args` and `kwargs` : https://docs.jina.ai/chapters/docstring/docstring.html#args-and-kwargs

Reply:
Can u recheck?